include/swift/Basic/Semantics.def

```diff
@@ -0,0 +1,78 @@
+//===--- Semantics.def - Semantics Attribute Definitions -------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This macro is used to map the global definition name of a semantic attribute to its raw value.
```

Comment:
nit: wrap to 80 chars. See also a few of the definitions below and the header of Semantics.h.